tests/auth_tests/test_auth_backends.py

```diff
@@ -598,6 +603,20 @@ def test_type_error_raised(self):
             authenticate(username='test', password='test')
 
 
+class WrongBackendTest(TestCase):
+    """
+    A backend is ignored if it doesn't accept provided credentials as arguments
+    """
+    backends = ('auth_tests.test_auth_backends.WrongBackend', 'django.contrib.auth.backends.ModelBackend')
+
+    def setUp(self):
+        self.user1 = User.objects.create_user('test', 'test@example.com', 'test')
+
+    @override_settings(AUTHENTICATION_BACKENDS=backends)
+    def test_skip_wrong_backend(self):
+        self.assertTrue(self.client.login(username='test', password='test'))
```

Comment:
No, this verifies that a backend is ignored if it doesn't accept provided credentials as arguments